.github/workflows/ci.yml

```diff
@@ -8,9 +8,46 @@ on:
     branches: [develop]
 
 jobs:
-  build:
+  quicktests:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+      - name: set up go
+        uses: actions/setup-go@v5
+        with:
+          check-latest: true
+          go-version-file: "go.mod"
+      - name: fmt, tidy
+        run: |
+          make install
+          make test-fmt
+          make test-tidy
+
+  lint:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+      - name: set up go
+        uses: actions/setup-go@v5
+        with:
+          check-latest: true
+          go-version-file: "go.mod"
+      - name: setup env
+        run: make install
+      - name: lint
+        run: make lint-github-action
```

Comment:
How about merging quicktests and lint into a single job?

Reply:
We could I wanted to keep the same structure as we have in other repos

Reply:
Sure, we can merge it everywhere :P Or leave it as is :shrug:

Reply:
For now I would leave it as it is. We can in the future define our own standardized jobs/steps in a dedicated repo and just import them into all others to get rid of this "code duplication" 🙂